frontends/web/package.json

```diff
@@ -58,7 +58,7 @@
   },
   "scripts": {
     "webpack": "webpack",
-    "start": "HTTPS=true SSL_CRT_FILE=/home/ubuntu/.ssl/dynabench.org.crt SSL_KEY_FILE=/home/ubuntu/.ssl/dynabench.org-key.pem react-scripts start",
+    "start": "HTTPS=true SSL_CRT_FILE=~/.ssl/local-cert.crt SSL_KEY_FILE=~/.ssl/local-cert.key react-scripts start",
```

Comment:
Let's not commit this - the prod server won't be able to find its certificates.

Reply:
I fixed this in the documentation btw, see https://github.com/facebookresearch/dynabench/commit/d5b9c56928b3b0513ee09f0741fee48f58e4c6a1

Reply:
Got it, I'll change this back to how it was originally.